app/react/translations/edit/editors/HTMLEditor/index.js

```diff
@@ -26,8 +35,8 @@ export default class HTMLEditor extends React.Component {
   state: StateTypes
   props: PropTypes
 
-  handleSubmit = (event: Event) => {
-    event.preventDefault();
+  handleSubmit = (e: Event) => {
+    if (e) e.preventDefault();
```

Comment:
Event is required. It'll be better to always propagate Event to this method, then checking if Event was provided.